documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -4,6 +4,16 @@
 [role="_abstract"]
 Configure {ispn_operator} to establish cross-site views with {brandname} clusters.
 
+[NOTE]
+====
+Cross-site replication does not support the {k8s} CA service, so you must choose one of the following options:
+
+* Create a certificate and then add it to each client that connects to a {brandname} endpoint.
+* Use an online service to upload the certificate to your server.
```

Comment:
I'm not sure what this means?

Reply:
Removed these bullets points based on your shortened note suggestion.